frame/example-offchain-worker/src/lib.rs

```diff
@@ -461,16 +461,6 @@ impl<T: Trait> Module<T> {
 		// Note this call will block until response is received.
 		let price = Self::fetch_price().map_err(|_| "Failed to fetch price")?;
 
-		// Received price is wrapped into a call to `submit_price_unsigned` public function of this
-		// pallet. This means that the transaction, when executed, will simply call that function
-		// passing `price` as an argument.
-		let call = Call::submit_price_unsigned(block_number, price);
-
-		// Now let's create a transaction out of this call and submit it to the pool.
-		// Here we showcase two ways to send an unsigned transaction with a signed payload
-		SubmitTransaction::<T, Call<T>>::submit_unsigned_transaction(call.into())
-			.map_err(|()| "Unable to submit unsigned transaction.")?;
-
 		// -- Sign using any account
 		let (_, result) = Signer::<T, T::AuthorityId>::any_account().send_unsigned_transaction(
 			|account| PricePayload {
```

Comment:
@tomusdrw 
But reading the code carefully, the `call` and `submit_unsigned_transaction` in L467 and L471 are just sending an unsigned tx. In this function, what we want to demo is doing unsigned tx with signed payload, which is code from L477 to L485. And they are not any way related to L467 - L472. Same logic applied for L503 - L512.

Probably it is an oversight that got copied over from L432 - L446? That's why I am removing them. Tell me if this is not the case.

Reply:
Fair enough! I thought that there is only one function that sends both a "raw" unsigned transaction and an unsigned transaction with signed payload. Thanks for spotting that this is a copy-paste error most likely.